AllJoyn/Platform/Bridge/BridgeNET/BridgeNET.vcxproj

```diff
@@ -190,13 +190,14 @@
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
```

Comment:
#Resolved

---
In reply to: [75174348](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75174348) [](ancestors = 75174348)